test/value.cpp

```diff
@@ -1583,6 +1583,38 @@ class value_test
             v3.value().get_string() == "value");
         BOOST_TEST(std::memcmp(
             v3.key_c_str(), "key\0", 4) == 0);
+
+        BOOST_STATIC_ASSERT(std::tuple_size<key_value_pair>::value == 2);
+        BOOST_TEST(get<0>(v2) == "key");
+        BOOST_TEST(
+            get<1>(v2).get_string() == "value");
+
+#if __cplusplus >= 201703L
```

Comment:
This won't fly on MSVC

Reply:
Good to know! Thanks for pointing it out; I believe I have now fixed that